### DISCUSSION TITLE
Initial View

### DISCUSSION THREAD

**Discussion by ghost** [Stellarium/stellarium#1996] [Ideas]
The fine program stellarium has been 'installed' in version 0.21.2 as appimage. When starting it within the initial view there is a small field in the lower left corner with the content "Erde [= earth in German], Cologne, 0 m". This I don't like for two reasons:

  - Cologne = Köln in German is a large city roughly 200 km north of Darmstadt where I live
  - "Cologne" is English, "Erde" is German, i.e. a language mixture

So I wanted to correct that, i.e. the coordinates of Darmstadt instead the ones of Cologne and a strict language choice.

BUT I could not find how to do that: pressing 'F1' shows a lot of helpful adjustments for the **daily** use of Stellarium, but **not** how to make **initial** adjustments 'once and for all'. 

So I suggest either

  - prolonging the F1-menue list correspondingly or
  - having at the end of this list a link somewhat like 'Initial adjustments'

As of now the corresponding place or link is hard to find; as a matter of fact I did not succeed.

A. Mehl

**Comment by gzotti** [comment]
Location names are English only. 
Please see the buttons in the left slideout menu. The thing looking like a compass rose is for setting locations. The thing looking like a clock is for adjusting time. 

Please read the beginning of the User Guide for more usage information.
